src/components/ItemsList.js

```diff
@@ -1,38 +1,77 @@
 import React from 'react';
 import firebase from '../lib/firebase';
 import Nav from './Nav';
+import '../styles/ItemsList.css';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 const db = firebase.firestore().collection('shopping_list');
 
+const wasItemPurchasedWithinLastOneDay = (lastPurchasedOn) => {
+  const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
+  return Date.now() - lastPurchasedOn <= oneDayInMilliseconds;
```

Comment:
This logic is great! So easy to follow and concise

Reply:
Thanks